pydatastructs/graphs/algorithms.py

```diff
@@ -700,6 +700,8 @@ def shortest_paths(graph: Graph, algorithm: str,
         'bellman_ford' -> Bellman-Ford algorithm as given in [1].
 
         'dijkstra' -> Dijkstra algorithm as given in [2].
+
+        'queue_improved_bellman_ford' -> Queue Improved Bellman-Ford algorithm as given in [3].
```

Comment:
Instead of a new one, let's just use Queues in Bellman Ford algorithm itself? Any specific for not doing so?

Reply:
If Queue version is always better then use Queue only. What's the difference between running times with and without queues?

Reply:
So let's always use a queue. Change the original bellman ford implementation.